recipe/meta.yaml

```diff
@@ -1,29 +1,29 @@
 {% set name = "sox" %}
 {% set version = "14.4.2" %}
+{% set commit = "b9dd1a86e71bbd62221904e3e59dfaa9e5e72046" %}
```

Comment:
Can we get those changes as a patch? It is more transparent that way in case the repo upstream changes.

Reply:
Updated that, I added a comment to the patch for the source of the visual studio fixes to maintain transparency there.